infra/base-images/base-builder/install_go_CodeIntelligenceTesting.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -eux
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Install CodeIntelligenceTesting Go.
+## Require at least Go1.4 to boostrap.
+cd /tmp
```

Comment:
I don't really like this pattern of cding into new directories and then using relative paths.
I would eliminate line 20 and change 21 to:
```
git clone --depth=1 -b dev.libfuzzer.18 https://github.com/CodeIntelligenceTesting/go.git /tmp/go-CodeIntelligenceTesting
```
and I would eliminate line 22 and change line 24 to:
```
sed -i '/^exec .* tool dist test -rebuild "$@"/ s/./#&/' /tmp/go-CodeIntelligenceTesting/run.bash
```

Reply:
Fixed in [here](https://github.com/google/oss-fuzz/pull/7240/commits/23beff81654b1ccaeae1eda7cc56d8fa76d933ac).